services/core-web/src/components/mine/Incidents/MineIncident.tsx

```diff
@@ -19,35 +19,46 @@ import { clearMineIncident } from "@common/actions/incidentActions";
 import * as Strings from "@common/constants/strings";
 import * as FORM from "@/constants/forms";
 import Loading from "@/components/common/Loading";
-import customPropTypes from "@/customPropTypes";
+// import customPropTypes from "@/customPropTypes";
```

Comment:
Please remove this commented line of code.

Reply:
Removed commented line and unused imports.